scripts/bench_all.py

```diff
@@ -2,9 +2,9 @@
 import bench_sorted
 import bench_hashes
 
-n_thread = 48
-sz_value = 200                          # About 256B/kv
-t_duration = 30                         # For operations other than fill
+n_thread = 64
+sz_value = 120                          # About 256B/kv
+t_duration = 10                         # For operations other than fill
```

Comment:
10 seconds is too short

Reply:
a long testing may exhausting pmem space